src/AppBundle/Document/DemoSeoExtractor.php

```diff
@@ -24,11 +24,7 @@
  *
  * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
  */
-class DemoSeoExtractor extends DemoSeoContent implements
-    TitleReadInterface,
-    DescriptionReadInterface,
-    OriginalUrlReadInterface,
-    KeywordsReadInterface
+class DemoSeoExtractor extends DemoSeoContent implements TitleReadInterface, DescriptionReadInterface, OriginalUrlReadInterface, KeywordsReadInterface
```

Comment:
rather disable this fixer and keep this multiline to make it readable. put `disabled: [single_line_class_definition]`  into .styleci.yml